src/main/groovy/actions/TransferActions.groovy

```diff
@@ -22,17 +23,20 @@ import actions.LabwareActions
 class TransferActions {
 
     def static stamp(Labware sourceLabware, Labware destinationLabware, 
-        MaterialType materialType, List<String> copyMetadata = []) {
+        MaterialType materialType, List<String> copyMetadata = [],
+        List<Metadatum> newMetadataToAdd = []) {
```

Comment:
OK, that was not clear from the acceptance criteria. It only says: add new metadata :(